huaweicloud/services/dli/resource_huaweicloud_dli_database_privilege.go

```diff
@@ -0,0 +1,244 @@
+package dli
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+const ErrCodeDbNotFound string = "DLI.0002"
```

Comment:
```suggestion
const errCodeDbNotFound string = "DLI.0002"
```

Reply:
acceptance also used, it is another package's file, out side this one